website/static/js/accessRequestManager.js

```diff
@@ -22,8 +22,8 @@ var AccessRequestModel = function(accessRequest, pageOwner, isRegistration, isPa
 
     self.permissionText = ko.observable(self.options.permissionMap[self.permission()]);
 
-    self.visible = ko.observable(true);
-
+    self.is_curator = ko.observable(accessRequest.user.is_curator || false);
+    self.visible = ko.observable(!accessRequest.user.is_curator);
```

Comment:
Could we make this visible but disabled to match with the other checkboxes in this control?

Reply:
You misunderstand, `self` is the user, not the HTML element, and visible refers to the user's bibliographic status. This doesn't effect the visibility of the checkboxes, it ensures all curators default to non-visible/bibliographic status.

Reply:
Gotcha. Pretend like this comment was down below in the mako file then.

Reply:
Yes I see that's a better visual user-facing move. I don't think Product will mind this slight deviation.